src/plugins/unified_search/public/search_bar/search_bar.tsx

```diff
@@ -364,7 +364,7 @@ class SearchBarUI extends Component<SearchBarProps, State> {
           isLoading={this.props.isLoading}
           fillSubmitButton={this.props.fillSubmitButton || false}
           prepend={
-            this.props.showFilterBar && this.state.query
```

Comment:
This was removed to be able to show saved query management, without filter bar. Filter bar added manually, omitting some features. Would it be safe change?